### PR TITLE
Update ViewController.swift

### PR DESCRIPTION
**Pull request by Fallenstedt**
With release of cocoapod Firebase 4.1.1, FIRAnalytics has been renamed to Analytics. withName is now an extraneous argument.

### CODE REVIEW

**Comment by Fallenstedt** [comment]
I signed it!

**Comment by samtstern** [comment]
cc @morganchen12

File: ios/swift/FirebaseAnalyticsWeb/ViewController.swift [hunk 1]
```diff
@@ -52,19 +52,20 @@ class ViewController: UIViewController, WKScriptMessageHandler {
     self.webView.load(request)
   }
 
+
   // [START handle_messages]
   func userContentController(_ userContentController: WKUserContentController,
-                             didReceive message: WKScriptMessage) {
+                           didReceive message: WKScriptMessage) {
```
Comment (morganchen12) [inline ios/swift/FirebaseAnalyticsWeb/ViewController.swift:58]: Please fix the inconsistent spacing in this file.

**Review by morganchen12 — CHANGES_REQUESTED** [review]
Looks good aside from a few nits. Thanks for doing this!

**Review by Fallenstedt — COMMENTED** [review]
I believe I have fixed the spacing.

**Comment by morganchen12** [comment]
Thanks!

@samtstern it seems I don't have push access, can you merge this and/or add me?

**Comment by Fallenstedt** [comment]
Bumping. It's been 13 days.

**Comment by samtstern** [comment]
Oh wow this is totally my fault!  I did not see that I was blocking this.  Thanks @Fallenstedt for bumping and sorry for the delay.

**Pull request merged**